rules/game_process.tex

```diff
@@ -75,6 +78,8 @@ \subsection{Robot States}
   \label{fig:robot_states}
 \end{figure}
 
+The referee must tell the GameController operator to switch to \texttt{initial} state once everyone is in place or that teams are falling under forfeit rule (see \cref{sec:forfeit}).
```

Comment:
> once everyone is in place or that teams are falling under forfeit rule (see \cref{sec:forfeit}).

I don't like introducing "once everyone is in place" here.

The head ref starts the game according to the schedule. If a team is not ready in time its their own fault. What happens if you are late/no-show is already detailed elsewhere.

Reply:
I didn't like either. I will try to formulate that differently.